src/client/tests/SessionTest.cpp

```diff
@@ -190,6 +191,24 @@ TEST_F(SessionTest, SSL) {
   EXPECT_TRUE(verifyResultWithoutOrder(*result.data, expected));
 }
 
+TEST_F(SessionTest, JsonResult) {
+  nebula::ConnectionPool pool;
+  nebula::Config c{10, 0, 10, 0, "", false};
+  pool.init({kServerHost ":9669"}, c);
+  auto session = pool.getSession("root", "nebula");
+  ASSERT_TRUE(session.valid());
+
+  auto resp = session.executeJson("YIELD 1");
+  folly::parseJson(resp);
+
+  folly::Baton<> b;
+  session.asyncExecuteJson("YIELD 1", [&b](std::string&& asyncResp) {
+    folly::parseJson(asyncResp);
+    b.post();
+  });
+  b.wait();
+}
+
```

Comment:
Add error test like `YIELDD 1` and retrieve the error msg from the json object

Reply:
This is not the client tests. The content of json result is generated by server, so server should test content of result.

Reply:
So we just retrieve the json and check it's valid json.